R/WebApi.R

```diff
@@ -142,3 +142,99 @@ getCdmSources <- function(baseUrl) {
 
   do.call(rbind, sourceDetails)
 }
+
+
+
+
+#' Retrieve the details of all Atlas definitions, by atlas functional category.
+#'
+#' @details
+#' Obtains the details of Atlas study definitions such as id, name, created/modified details, hash object, etc.
+#' The following atlas function categories are supported. Concept-set, Cohort-definition,
+#' Cohort-characterization, Pathway-analysis, Incidence rate (ir), estimation and prediction.
+#' This function is useful to retrieve the current definitions in one atlas instance and comparing it
+#' to another atlas instance, or for version control.
+#'
+#'
+#' @param baseUrl   The base URL for the WebApi instance, for example: "http://server.org:80/WebAPI".
+#'
+#' @return
+#' A data frame of atlas study definitions with details. Note: modifiedDate and createdDate are
+#' returned as text/character (to be worked on in future version).
+#'
+#' @export
+getAtlasDefinitionDetails <- function(baseUrl) {
```

Comment:
I would think 'getAtlasDefinitionsDetails`? You're very likely getting details for more than one definition.